src/pages/GamesList.js

```diff
@@ -35,8 +35,8 @@ export class FutureGamesList extends React.Component {
   state = {
     loading: true,
     games: null,
-    displayEmptyGames: false,
-    displayFullGames: false,
+    displayEmptyGames: ( sessionStorage.getItem('selector_displayEmptyGames') === 'true' ? true : false ),
```

Comment:
Since we already have it in multiple places done similar way, maybe we can provide utility function for getting and setting values?

Also it can be written shorter as: 
```javascript
displayEmptyGames: sessionStorage.getItem('selector_displayEmptyGames') === 'true' // returns true or false
```

To avoid storing bool values as strings, it may be a good idea to store them as 1 or 0 and cast to Boolean:
```javascript
displayEmptyGames: Boolean(sessionStorage.getItem('selector_displayEmptyGames')) || false
```

It's up to you which you like more :) Anyway I think it may be a good improvement for the future use of code if we have one utility function for setting such local settings as true/false values.